### ISSUE TITLE
Extracting confidence

### ISSUE THREAD

**Issue by naveenjafer** [Vin-Yuan/CNN_sentence#1]
@Vin-Yuan  
Hi,
   I see that you have used the softmax function to retrieve metrics in the code. If i were to obtain the confidence value of an input test sentence for a class, what tensorflow function should I be using? I am new to tensorflow so excuse me for the lack of knowledge.
